### ISSUE TITLE
Version 5 for 2021.2p2 doesn't support Mojave

### ISSUE THREAD

**Issue by austinhuang0131** [Free-GPGMail/Free-GPGMail#36]
Compatibility Marker

But version 5 for 2021.2 does not have this issue

**Comment by bnavigator** [comment]
There has been no change in the compatibility markers of Free-GPGMail 5 since 2021.2

**Comment by austinhuang0131** [comment]
![Screen Shot 2021-12-26 at 13 15 39](https://user-images.githubusercontent.com/16656689/147416704-2e37bfad-15e6-4a90-9202-28c1d23e95ba.png)

Just verified again

**Comment by bnavigator** [comment]
Please type these two commands in your Terminal and post the output:

```bash
sw_vers -productVersion
defaults read /System/Applications/Mail.app/Contents/Info PluginCompatibilityUUID
```

**Comment by austinhuang0131** [comment]
```bash
$ sw_vers -productVersion
10.14.6
$ defaults read /System/Applications/Mail.app/Contents/Info PluginCompatibilityUUID
2021-12-26 13:19:21.590 defaults[49499:8504174] 
The domain/default pair of (/System/Applications/Mail.app/Contents/Info, PluginCompatibilityUUID) does not exist
```

**Comment by bnavigator** [comment]
Ah, Mojave might it have under

`defaults read /Applications/Mail.app/Contents/Info PluginCompatibilityUUID`

**Comment by austinhuang0131** [comment]
```
$ defaults read /Applications/Mail.app/Contents/Info PluginCompatibilityUUID
A4343FAF-AE18-40D0-8A16-DFAE481AF9C1
```

**Comment by bnavigator** [comment]
This UUID is present in Free-GPGMail_5 2021.2p2

**Comment by bnavigator** [comment]
Did you disable or delete the Mailloader?

**Comment by austinhuang0131** [comment]
What mailoader? If you mean the bundle, then I replace the file and then enable it, which produces the message. I also use another mailbundle plugin (MailTrackerBlocker on GitHub).

**Comment by austinhuang0131** [comment]
Persists in 2021.3

**Comment by bnavigator** [comment]
Fixed in 2021.3.1
